### PR TITLE
Most used tags: Try fixing label

### PR DESCRIPTION
**Pull request by jasmussen**
## What?

Most used tags is a thin weight:

Screenshot 2022-10-11 at 09 20 18

I think it was designed for a previous style of labels that weren't small and uppercase. This PR just removes the font weight:

Screenshot 2022-10-11 at 09 21 08

The prepublish interface was also missing the style entirely:

Screenshot 2022-10-11 at 09 23 12

I made a biggish change to the code that styles the default inspector, to also encompass the pre-publish sidebar. I'll comment in the code and ask for an extra look:

Screenshot 2022-10-11 at 09 29 02

## Testing Instructions

Have at least 4 different labels, and 3 of them used on a bunch of posts. Then observe the Tags sidebar, and the pre publish tag suggestion dialog.

### CODE REVIEW

File: packages/interface/src/components/complementary-area/style.scss
Comment (jasmussen) [inline packages/interface/src/components/complementary-area/style.scss]: So this is obviously a big change, given it's SCSS. But at the moment, it targets the entire inspector, why shouldn't it target also the pre-publish inspector? Would love a sanity check on this one. Thank you!

**Review by Mamaduka — APPROVED** [review]
The fixes look great. Thank you, Joen!

I can't think of why complementary area styles shouldn't target the per-publish inspector. The panel designs are mostly identical.

Let's see if others have any objections to this change.

**Comment by jasmussen** [comment]
Thanks. I'll let this one sit for others to chime in on, since it isn't particularly urgent.

File: packages/interface/src/components/complementary-area/style.scss
Comment (mirka) [reply packages/interface/src/components/complementary-area/style.scss]: Just speaking from a general CSS architecture standpoint, this kind of thing isn't encouraged because it breaks component encapsulation. Styles for `InterfaceSkeleton` should not live in a different component's folder.

https://github.com/WordPress/gutenberg/blob/db3b6478ae8254252c9848094bb88761a963bc85/packages/interface/src/components/complementary-area/style.scss#L44-L51

And assuming that the reason you want to do this is to get these uppercase styles ☝️, this kind of styling (targeting all `h3`s) also isn't generally tenable in a componentized app. 

So my quick fix suggestion for this specific issue would be to change this:

https://github.com/WordPress/gutenberg/blob/db3b6478ae8254252c9848094bb88761a963bc85/packages/editor/src/components/post-taxonomies/most-used-terms.js#L50-L52

to:

```jsx
<BaseControl.VisualLabel as="h3" className="editor-post-taxonomies__flat-term-most-used-label"> 
  { label } 
</BaseControl.VisualLabel> 
```

so we can get the label styles cleanly. Let me know if you need further assistance, and I can push a commit to this branch if you would like.

Comment (jasmussen) [reply packages/interface/src/components/complementary-area/style.scss]: Please feel free to push to this branch!

Thank you.

**Review by mirka — APPROVED** [review]
I pushed up my suggestions. Good to go on my end if it still does what you intended to do initially 👍

**Comment by jasmussen** [comment]
It does! Nice, thank you!

Screenshot 2022-10-14 at 13 37 14

Screenshot 2022-10-14 at 13 37 19

**Pull request merged**
